pkg/daemon/ceph/osd/volume.go

```diff
@@ -149,12 +153,28 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 		// Reporting
 		reportArgs := append(batchArgs, []string{
 			"--report",
+			"--format",
+			"json",
 		}...)
 
-		if err := context.Executor.ExecuteCommand(false, "", baseCommand, reportArgs...); err != nil {
+		cvOut, err := context.Executor.ExecuteCommandWithOutput(false, "", baseCommand, reportArgs...)
+		if err != nil {
 			return fmt.Errorf("failed ceph-volume report. %+v", err) // fail return here as validation provided by ceph-volume
 		}
 
+		logger.Infof("ceph-volume report: %+v", cvOut)
```

Comment:
I think this should be a `Debugf` rather than info.

Reply:
I personally like having the c-v report output logged prior to provisioning the OSDs so if your OSD configuration does not match your expectations you can check the log and see why.  Otherwise you would have to clean up the OSDs, enable DEBUG logging and then provision again to get the report.  I also left the report off of the error message since it was just printed here.  If you do still want this to be a debug message I am happy to change it and add the report to the error message.

Reply:
I'm on the fence, so I will leave the choice up to you as a user. There is a balance to be made between overly verbose logs and output that is beneficial to a large number of users. I try to keep in mind that often, restarting the operator with debug logging on is sufficient to figure out an error after it has occurred. As well, overly verbose versus beneficial may change for clusters with dozens of nodes.

Reply:
@mvollman The report is already printing in a human readable form and i thought it was working well. Why do we need to change it to json? It seems less readable to the human eye. If we need the json report in order to determine the error condition below, perhaps we should run the readable report first, then run the report again to get the json, but not print the json to the log

Reply:
Agreed, the human readable report is nicer to look at.  I was just trying to avoid running ceph-volume 3 times, but I can certainly change it to print the human readable report instead.